src/services/donations/donations.hooks.js

```diff
@@ -92,6 +92,26 @@ const poSchemas = {
   },
 };
 
+const donationResolvers = {
+  joins: {
+    token: () => async (donation, context) => {
+      const { tokenAddress } = donation;
+      const token = getTokenByAddress(tokenAddress);
+      if (token) {
+        donation.token = token;
+      }
+    },
+  },
+};
+
+const convertTokenToTokenSymbol = () => context => {
```

Comment:
Remove this function, we should manually remove token entities later by some script